css-canvas-medical/styles.css

```diff
@@ -0,0 +1,243 @@
+.container {
+  margin: auto;
+  max-width: 800px;
+  padding-top: 20px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.column-one-fifth {
+  flex-basis: calc(100%/5);
+  font-family: Avenir, sans-serif;
+  font-size: 13px;
+  padding-top: 22px;
+  color: rgb(185 185 185);
```

Comment:
Only `flex-basis` should be applied to `column-` classes. Make another class that you can add to the element in addition to the `column-one-fifth` class.